src/client/components/HomepageHeader/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import robot from '../../assets/images/robot.png';
+import './header.css';
+import PropTypes from 'prop-types';
+
+function Header({ color, size }) {
+  return (
+    <header className="homepage-header">
+      <h2 style={{ color: `${color}`, fontSize: `${size}px` }}>
+        What is GuideIT?
+      </h2>
+      <img className="robot-image" src={robot} alt="robot" />
+    </header>
+  );
+}
+
+Header.propTypes = {
+  color: PropTypes.string.isRequired,
+  size: PropTypes.number.isRequired,
+};
+
+export default Header;
```

Comment:
The Header looks super nice to me :) @sandhana14.
But I wonder, is it necessary to use props as I was thinking we would not need to change the header color or font-size somewhere in the app. Rather it should be static.

Reply:
Hi Tithi. Thank you for your review. I felt the same but I wanted to give some flexibility that's why I tried.